### PR TITLE
MINOR: Fixes broken build

### PR DESCRIPTION
**Pull request by jlprat**
Because of lack of implicit conversions, boolean properties need to be passed as Strings
This is done in other parts of the code already

Fixes broken build introduced in https://github.com/apache/kafka/pull/14846

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by mimaison — APPROVED** [review]
LGTM, I confirm trunk builds fine with Scala 2.12 with this

**Comment by jlprat** [comment]
I'll wait until the build finishes and then I'll merge

**Review by nizhikov — APPROVED** [review]

**Review by tinaselenge — APPROVED** [review]

**Comment by mimaison** [comment]
@jlprat If you've validated the test this changes works, we probably don't need to wait the few hours for the build to complete to merge.

**Comment by jlprat** [comment]
I confirmed this change fixes the broken build for Scala 2.12, hence I merged before the CI finished.

**Comment by dengziming** [comment]
Thank you @jlprat for the fix!

**Pull request merged**
